app/Http/Controllers/HomeController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class homecontroller{ $th }
```

Comment:
__[code_sniffer]__
* Class name "homecontroller" is not in camel caps format
* Opening brace of a class must be on the line after the definition
* There must not be more than one property declared per statement
* Visibility must be declared on property "$th"
* The closing brace for the class must go on the next line after the body
* Closing brace must be on a line by itself

<!-- Comment by [SideCI](https://sideci.com) -->